src/System.Private.ServiceModel/tools/scripts/SetupWcfIISHostedService.cmd

```diff
@@ -169,18 +203,22 @@ goto :Done
 echo.
 echo Setup WCF test services hosted on IIS for the testing of WCF on .NET Core
 echo.
-echo Usage: %~n0 Id [/c]
+echo Usage: %~n0 Id [/c] [/p:{PathToWcfRepo}]
 echo    Id: The Id of a WCF repo and its associated IIS hosted services to be created.
 echo        The Id will be appended to the name of all assets to be created.
 echo        %_idMaster%: if Id is '%_idMaster%', additional IIS hosted services such as PRService
 echo        will be created to serve as central services for other WCF repos. 
 echo    /c: If specified, this will clean up any existing setup of Id instead of creating new.
+echo    /p: If an existing WCF repo is preferred to be used, use this parameter to provide
+echo        the path to a WCF repo.
```

Comment:
Nit: optionally add to this explanation what will happen if /p is not provided.  In other words, what is the default?